docs/examples/google-cloud-function.ts

```diff
@@ -0,0 +1,29 @@
+import { Telegraf } from 'telegraf'
+
+const { BOT_TOKEN, PROJECT_ID, FUNCTION_NAME, REGION } = process.env
+
+// eslint-disable-next-line
+const bot = new Telegraf(`${BOT_TOKEN}`, {
+  telegram: { webhookReply: true },
+})
+
+// eslint-disable-next-line
+bot.telegram.setWebhook(
+  // eslint-disable-next-line
+  `https://${REGION}-${PROJECT_ID}.cloudfunctions.net/${FUNCTION_NAME}`
+)
+
+bot.command('hello', (ctx) => ctx.reply('Hello, friend!'))
+
+export const botFunction = async (req: any, res: any) => {
+  console.log(req.body)
+  try {
+    await bot.handleUpdate(req.body)
+    console.log('Success')
+    res.status(200).send('Success')
+  } catch (err) {
+    console.log('Failure')
+    console.log(err)
+    res.status(500).send('Something went wrong')
```

Comment:
This means that Telegram will resend the update and probably cause the same error again, right? Why is that desirable?

Reply:
No, the catch is to handle calls to the cloud function that are not from telegram

Reply:
How do you expect this to work? I'd expect you to check request path instead.

Please don't mark a conversation as resolved that I unresolved.
Also, I'll probably be inactive here for a while, focusing on 4.1.0 release. This will ship in 4.1.1.

Reply:
I can just remove the error checking if the way I'm handling it isn't okay. Most of the other examples here don't have any

Reply:
I'm unfamiliar with the platorm, but I'd do something akin to

```ts
try {
  await bot.handleUpdate(req.body)
finally {
  res.status(200)
}
```

Then if an update causes the bot to error, it won't be redelivered and won't cause another error.

Reply:
Updated